src/mixer/playermanager.cpp

```diff
@@ -188,12 +188,17 @@ void PlayerManager::bindToLibrary(Library* pLibrary) {
     // analyzed.
     foreach(Deck* pDeck, m_decks) {
         connect(pDeck, &BaseTrackPlayer::newTrackLoaded, this, &PlayerManager::slotAnalyzeTrack);
+        connect(pDeck, &BaseTrackPlayer::trackUnloaded, this, &PlayerManager::slotStoreParkedTrack);
     }
 
     // Connect the player to the analyzer queue so that loaded tracks are
     // analyzed.
     foreach(Sampler* pSampler, m_samplers) {
         connect(pSampler, &BaseTrackPlayer::newTrackLoaded, this, &PlayerManager::slotAnalyzeTrack);
+        connect(pSampler,
+                &BaseTrackPlayer::trackUnloaded,
+                this,
+                &PlayerManager::slotStoreParkedTrack);
     }
 
     // Connect the player to the analyzer queue so that loaded tracks are
```

Comment:
I decided not to connect the signal to the preview deck, because that deck will have a lot more churn, but that could be changed